TESTING.md

```diff
@@ -152,7 +153,49 @@ For example, the unit test run as a command before the acceptance test:
 ```
 testacc: testunit
 	@sh -c "'$(CURDIR)/scripts/runtest.sh' acceptance"
-``` 
+```
+
+
+## Binary testing
+
+By *binary testing* we mean the tests that run using Terraform binary executable, as opposed to running the test through the Go framework.
+This test runs the same tasks that run in the acceptance test, but instead of running them directly, they are fed to the
+terraform tool through a shell script, and for every test we run
+
+* `terraform init`
+* `terraform plan`
+* `terraform apply -auto-approve`
+* `terraform destroy -auto-approve`
+
+The test runs from GNUMakefile, using
+
+```bash
+make test-binary
+```
+
+All the tests run unattended, stopping only if there is an error.
+
+It is possible to run the tests with manual control, by preparing them and then running the test script from the `tests-artifacts` directory:
```

Comment:
"Manual control" is a bit unclear in this context (you start to thinking about pausing the tests), so I suggest this wording instead: "It is possible to customise running of the binary tests, by ..."

Reply:
changed